README_DEVELOPERS.md

```diff
@@ -43,3 +43,17 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
+
+## Run Updating Templates for Existing Applications
```

Comment:
I think that updating templates is a separate flow and should be documented in a separate section (not in `Testing`).

Reply:
Added section